system_monitoring_system/__main__.py

```diff
@@ -3,9 +3,37 @@
 import gui
 import monitoring
 import report
+import os
+import json
+import shutil
+
+
+def create_sms_folder():
+    path = os.path.join(os.path.expanduser('~'), '.sms')
+
+    try:
+        if not os.path.exists(path) or not os.path.isdir(path):
+            os.mkdir(path)
+            shutil.copy('./Montserrat-Regular.ttf', path)
+            shutil.copy('./Montserrat-Bold.ttf', path)
+            shutil.copy('./sms_icon.png', path)
+            shutil.copy('./Montserrat-Regular.ttf', path)
```

Comment:
Why are the statements getting repeated?

Reply:
Fixed.